nulib/digital-collections
# Ability to embed Audio/Sound works.

![image](https://user-images.githubusercontent.com/7376450/145608177-96090c6c-3374-4973-82bb-ce6432df68d0.png)

## Summary 
This pull request includes updates to allow works with **AUDIO** and **VIDEO** `workType` values to be embedded by the end user. The functionality extends the existing screen routed to the path `/embedded-viewer/...` and adds an optional workType param following the path of `/embedded-viewer/:manifestUrl/:workType?`. To make this backwards compatible with embeds that may have been shared prior to this param, the screen will default to `image` if this param is not set.

For workTypes of `audio` or `video` the React Media Player will render on screen. Works for `image` will continue to use React OpenSeadragon.

## Specific Changes in this PR
- New param of workType created for the embedded-viewer path
- A/V Works can now be embedded with the React Media Player
- Styling touch ups on the Download and Share tab. 
- For A/V works, auxiliary images in the IIIF manifest will be mapped to the Download/Embed table.

## Steps to Test

1. Spin up `npm run start:use-staging-data`
2. Filter by video, select Nimiipuu (this Video work has both Access and Aux image filesets)
3. Go to Download/Share tab
4. Things should look a little more tidy and two Aux images should display in table
5. Embed Code for Rich Viewer should render out a `src` ending in `.../video`
6. That `src should render the React Media Player
7. Repeat with any Image work and these should render the React OpenSeadragon viewer
